app/ide-desktop/eslint.config.js

```diff
@@ -137,14 +133,6 @@ const RESTRICTED_SYNTAXES = [
         selector: `TSAsExpression:not(:has(TSTypeReference > Identifier[name=const]))`,
         message: 'Avoid `as T`. Consider using a type annotation instead.',
     },
-    {
-        selector: `:matches(\
-            TSUndefinedKeyword,\
-            Identifier[name=undefined],\
-            UnaryExpression[operator=void]:not(:has(CallExpression.argument)), BinaryExpression[operator=/^===?$/]:has(UnaryExpression.left[operator=typeof]):has(Literal.right[value=undefined])\
-        )`,
-        message: 'Use `null` instead of `undefined`, `void 0`, or `typeof x === "undefined"`',
```

Comment:
these other two lints aren't related to this PR - if you want to remove them they should at least be related to the PR imo

Reply:
See: https://github.com/enso-org/enso/pull/10200#discussion_r1630713853

Reply:
it's not productive to discuss changes in a PR where the effects are not visible IMO

Reply:
Ok lets omit them